backends/chromecast/cast-utils.js

```diff
@@ -125,12 +125,14 @@ function cast(flags, log, mode, url) {
           // The request was fulfilled.
           log.info('Cast successful.');
           clearTimeout(connectionTimer);
+          client.close();
```

Comment:
Is it intended? Why do we need to close the client on launch?

Reply:
Even when the main script terminates, nodejs stays running because of an async method like this one.  But even when we call resolve() to end this Promise, nodejs **still** stays open until all sockets are closed.  So without this, it finishes work and just sits there for 30 more seconds until the socket times out and is closed.

Reply:
I added more info to the PR description.  Does this help?